docs/guides/create_fsm_app.md

```diff
@@ -0,0 +1,2 @@
+!!!! note
+    This section will be added soon.
```

Comment:
Don't create place holders like this, create A file when you plan on adding the complete documentation

Reply:
I am aware this is not ideal. However, I did this approach for 2 reasons:

- the PR filling this information is comming quickly, and I want to keep the PRs semantically "consistent" (i.e., this PR is not about FSM apps...)
- It facilitates writing links in currently working documents (otherwise, I have to revise the links after writing the section...)